config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -803,123 +803,3 @@ presubmits:
           limits:
             cpu: 1
             memory: "2Gi"
-
-  # TODO(jprzychodzen): Remove after deprecation of release-1.19
-  - name: pull-perf-tests-verify-all
-    decorate: true
-    always_run: true
-    skip_branches:
-    - release-1.1[2-8] # Presubmit has been implemented for 1.19+ releases
-    - master # It's covered by 'pull-perf-tests-verify-all-python' and 'pull-perf-tests-verify-test'
-    path_alias: "k8s.io/perf-tests"
-    annotations:
-      testgrid-dashboards: presubmits-kubernetes-scalability
-      testgrid-tab-name: pull-perf-tests-verify-all
-    spec:
-      containers:
-      - image: public.ecr.aws/docker/library/golang:1.20
-        command:
-        - make
-        args:
-        - verify-all
-        resources:
-          requests:
-            cpu: 1
-            memory: "2Gi"
-          limits:
-            cpu: 1
-            memory: "2Gi"
-
-  - name: pull-perf-tests-verify-all-python
-    decorate: true
-    always_run: true
-    skip_branches:
-      - release-1.1[2-9] # Presubmit has been implemented for 1.20+ releases
-    path_alias: "k8s.io/perf-tests"
-    annotations:
-      testgrid-dashboards: presubmits-kubernetes-scalability
-      testgrid-tab-name: pull-perf-tests-verify-python-all
-    spec:
-      containers:
-        - image: python:3
-          command:
-            - make
-          args:
-            - verify-all-python
-          resources:
-            requests:
-              cpu: 1
-              memory: "2Gi"
-            limits:
-              cpu: 1
-              memory: "2Gi"
-
-  - name: pull-perf-tests-verify-test
-    decorate: true
-    always_run: true
-    skip_branches:
-    - release-1.19 # We use verify-all for that branch
-    path_alias: "k8s.io/perf-tests"
-    annotations:
-      testgrid-dashboards: presubmits-kubernetes-scalability
-      testgrid-tab-name: pull-perf-tests-verify-test
-    spec:
-      containers:
-      - image: public.ecr.aws/docker/library/golang:1.20
-        command:
-        - verify/test.sh
-        resources:
-          requests:
-            cpu: 1
-            memory: "2Gi"
-          limits:
-            cpu: 1
-            memory: "2Gi"
-
-  - name: pull-perf-tests-verify-dashboard
```

Comment:
This presubmit runs for master and all previous supported versions (we can remove 'skip_branches' now).